lighthouse-core/scripts/update-report-fixtures.js

```diff
@@ -7,6 +7,8 @@
 
 const cli = require('../../lighthouse-cli/run.js');
 const cliFlags = require('../../lighthouse-cli/cli-flags.js');
+const assetSaver = require('../lib/asset-saver.js');
+const artifactPath = 'lighthouse-core/test/results/artifacts';
```

Comment:
```suggestion

const artifactPath = 'lighthouse-core/test/results/artifacts';
```

just add some whitespace after our requires